doc/2/framework/classes/plugin-context/properties/index.md

```diff
@@ -6,3 +6,98 @@ description: PluginContext class properties
 ---
 
 # PluginContext
+
+The `PluginContext` class is passed to the plugins as an argument of the [init](/core/2/some-link) method.  
```

Comment:
Considering that we're documenting frameworks and not plugin, I think that it would be clearer for users to rename this object "ApplicationContext".

A simple alias would do the trick for now, but I foresee ApplicationContext and PluginContext going different paths in the future.

Reply:
I'm not sure to understand because the `PluginContext` is meant to be used only in plugins. Application are using the `Backend` class to have a "context"